com.sudovanilla.penpot-desktop.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: com.sudovanilla.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://sudovanilla.com/distribute/applications/penpot-desktop/latest/Penpot-Desktop.AppImage
```

Comment:
how does this work on aarch64?

Reply:
No clue...why you think I had that restriction set?
Anyways, I'm don't own a arch64 Linux machine to build a arch64 executable.

Searching for solutions...

Reply:
I do have access to a Raspberry Pi 4 I can setup soon.

Reply:
An arm64 AppImage file has been built. However, how do I go about specifically telling the YAML file which is the x86_64 build what is the arm64 build? 

If there a proper documentation that would tell me this?